src/Microsoft.AspNet.OData.Shared/DeltaOfTStructuralType.cs

```diff
@@ -435,6 +443,19 @@ public TStructuralType CopyChangedValues(TStructuralType original)
                 throw Error.Argument(nameof(original), SRResources.DeltaTypeMismatch, _structuredType, original.GetType());
             }
 
+            //To apply ODataId based handler to apply properties on original.
```

Comment:
what does this statement mean? what is ODataId based handler?

Reply:
Refactored some logic and removed this code here and move it to `DeltaSet<T>`. I have added a proper comment there.